app/views/foods/index.html.erb

```diff
@@ -0,0 +1,31 @@
+<div class="row">
+  <a href="<%= new_food_path %>" class="btn btn-primary btn block col-sm-3 offset-sm-8">Add New</a>
+</div>
+<div class="row mt-5">
+  <div class="col-md-4 offset-md-4">
+    <table class="table">
+      <thead>
+        <tr>
+          <th>Food</th>
+          <th>Meausurement Unit</th>
+          <th>Unit Price</th>
+          <th>Action</th>
+        </tr>
+      </thead>
+      <tbody>
+        <% @foods.each do |food| %>
+          <tr>
+            <td><%= food.name %> </td>
+            <td><%= food.measurement_unit %> </td>
+            <td>$<%= food.price %> </td>
+            <td>
+              <% if can? :destroy, food  %>
+                <%= link_to "Delete", food_path(id: food.id), class:"btn btn-outline-danger", "data-turbo-method": :delete %></span>
```

Comment:
- [ ] The functionality to delete a food item is not working correctly. Please see the bug 🐛 encountered when the 'Delete' button is clicked 👇 
![image](https://user-images.githubusercontent.com/46839548/221331590-7ba25bde-51e1-47bc-82e2-1be39aa1b475.png)